fdbserver/DataDistribution.actor.cpp

```diff
@@ -4990,14 +4980,11 @@ ACTOR Future<Void> dataDistribution(Reference<DataDistributorData> self, Promise
 		}
 		catch( Error &e ) {
 			state Error err = e;
-<<<<<<< HEAD
 			TraceEvent("DataDistributorDestroyTeamCollections").error(e);
 			self->teamCollection = nullptr;
 			primaryTeamCollection = Reference<DDTeamCollection>();
 			remoteTeamCollection = Reference<DDTeamCollection>();
-=======
 			trackerCancelled = true;
```

Comment:
To be safe, I think `trackerCancelled` shoud be set to `true` before the team collections are cleared. This is the behaviour I tested on 6.3, and cancelling the team collections can trigger a bunch of callbacks with unexpected results.